File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int i = 0;
        int j = 1;
        int count = 0;
        while (i<nums.length && j<nums.length){
            if(nums[i]>nums[j]) count++;
            i++;
            j++;
        }
        return count<=1 && (count==0 || nums[0]>=nums[nums.length-1]);
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        count = 0
        if nums[-1]>nums[0]:
            count+=1
        for i in range(len(nums)-1):
            if nums[i]>nums[i+1]:
                count+=1
        if count<=1:
            return True
        return False

        